src/content/en/fundamentals/primers/service-workers/the-players.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: shared/narrow
+title: "The Players"
+description: "A basic Service Worker Implementation only has two players."
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 3
+translation_priority: 0
+authors:
+- josephmedley
+---
+
+<p class="intro">
+  A basic Service Worker Implementation only has two players.
+</p>
+
+ 
+**Clients**&mdash;These players install and use a service worker. Generally, they 
+are web pages. Though there are [other types of clients](https://developer.mozilla.org/en-US/docs/Web/API/Client)
+, this primer focuses on web page clients. The examples described in 
+[What do they Do?](what-do-they-do.markdown) are all web page clients.
```

Comment:
`[What do they Do?](what-do-they-do.markdown)`
should be
`[What do they Do?](what-do-they-do)`